Textfield/Textfield.jsx

```diff
@@ -85,7 +85,14 @@ class TextfieldInput extends MaterialComponent {
             />}
         {props.label &&
           this.state.showFloatingLabel &&
-          <Label for={props.id}>
+          <Label
+            for={props.id}
+            className={
+              this.state.showFloatingLabel
```

Comment:
If I'm not missing something, this condition is always true so this is actually exactly the same code as before this commit.

Reply:
even I thought so, so this will not be true in case of SSR(cDM works on client only r8?), hence an SSR'd html will have this class and avoid FOUC

Reply:
Yes but the condition to show the label is ```props.label && this.state.showFloatingLabel```, so in this case ```this.state.showFloatingLabel``` must be true, right?
In case of SSR we show the other label, the one in ```Textfield``` and not ```TextfieldInput```. Maybe this means we can't avoid FOUC?

Reply:
>Yes but the condition to show the label is props.label && this.state.showFloatingLabel, so in this case this.state.showFloatingLabel must be true, right?

Oh yes silly me.

lets revert my last commit and close publish this, also lets open another issue for FOUC